src/modules/Search/Search.js

```diff
@@ -508,11 +516,12 @@ export default class Search extends Component {
   // ----------------------------------------
 
   renderSearchInput = () => {
-    const { icon, placeholder } = this.props
+    const { icon, placeholder, input } = this.props
     const { value } = this.state
 
     return (
       <Input
+        fluid={input.fluid}
```

Comment:
Hm, was this tested?  The default props should be merged, not replaced.

https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/lib/factories.js#L84